### ISSUE TITLE
Add u.replace or something to replace the nested object

### ISSUE THREAD

**Issue by boatkorachal** [substantial/updeep#19]
for example i have the object
```javascript
let obj = {
    objToReplace: {
        key1: 'val1',
        key2: 'val2'
    }
}
```

currently if I want to replace the object at key 'objToReplace' with new object, I have to update with function.
```javascript
let replacedObj = u({
    objToReplace: () => ({
        key3: 'val3'
    })
}, obj)

// replacedObj = {
//     objToReplace: {
//         key3: 'val3'
//     }
// }
```

Do you think is it good to add the shortcut for this kind of operation?

Thanks!
(sorry for my bad english)

**Comment by aaronjensen** [comment]
I do! See https://github.com/substantial/updeep/issues/10




I didn't mention the use case but it is exactly this. 



--

SUBSTANTIAL  |  Aaron Jensen  |  Developer  |  OFFICE: 206‑838‑0303  |  MOBILE: 805‑441‑9288

**Comment by boatkorachal** [comment]
oh! that is awesome! :)

**Comment by aaronjensen** [comment]
Closing this and tracking in #10 

Feel free to submit a pull request if you want to give this a shot! Thanks :smile:

**Comment by aaronjensen** [comment]
Fixed in 0.7.0
